src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_client_factory.py

```diff
@@ -44,76 +43,69 @@ def storage_client_factory(cli_ctx, **_):
 
 
 def file_data_service_factory(cli_ctx, kwargs):
-    FileService = get_sdk(cli_ctx, ResourceType.DATA_STORAGE, 'file#FileService')
-    return generic_data_service_factory(
-        cli_ctx, FileService,
-        kwargs.pop('account_name', None),
-        kwargs.pop('account_key', None),
-        connection_string=kwargs.pop('connection_string', None),
-        sas_token=kwargs.pop('sas_token', None))
+    t_file_svc = get_sdk(cli_ctx, ResourceType.DATA_STORAGE, 'file#FileService')
```

Comment:
Why use `t_file_svc` instead of the original name of `FileService`?
Same everywhere else.

Reply:
In my opinion this is a better name convention. The original one is misleading. The reader is not able to tell if it is a type actually defined in source code or a type from the SDK. Alternatively, we can adopt this pattern: https://www.python.org/dev/peps/pep-0008/#type-variable-names (of course it makes your code very "C" like).

Reply:
ok fair